pkgsrc/dev_compile_script.sh

```diff
@@ -1,3 +1,4 @@
+set -eux
 export BUILD_FOLDER=/tmp
```

Comment:
it breaks some things when you re-run the script. I've prepared a fix for this + some other things and i'll send a PR when they are ready for upstreaming.